Description: A pod2man error in backup-manager-purge has been fixed.
Author: Sven Joachim <svenjoac@gmx.de>

---
 backup-manager-purge |    2 ++
 1 file changed, 2 insertions(+)

--- a/backup-manager-purge
+++ b/backup-manager-purge
@@ -53,6 +53,8 @@
 
 =head1 OPTIONAL ARGS
 
+=over 4
+
 =item B<--files-from=>file
 
 A file containing a list of archives to parse, one archive per line.
